eng/Build.ps1

```diff
@@ -145,7 +143,7 @@ function Process-Arguments() {
 }
 
 function Update-Arguments() {
-    if (-Not (Test-Path "$ArtifactsDir\Bootstrap\fsc\fsc.exe")) {
+    if (-Not (Test-Path "$ArtifactsDir\Bootstrap\fsc.exe")) {
```

Comment:
The different applications in the bootstrap tools should be in different directories, since they may have different dependencies